### DISCUSSION TITLE
Should i use NewClient or NewClusterClient with aws serverless Redis oss?

### DISCUSSION THREAD

**Discussion by cy724** [redis/go-redis#3187] [General]
I am using [AWS Redis OSS serverless](https://docs.aws.amazon.com/AmazonElastiCache/latest/dg/supported-engine-versions.html#redis-version-7.1)
<img width="2455" alt="image" src="https://github.com/user-attachments/assets/9fe760d0-85d3-4901-a641-5b59abb6ca91">
There is only one endpoint, Should i use [redis.NewClient](https://github.com/redis/go-redis/blob/80c9f5bb777dd4e6393d014cffe9d28428ecf756/redis.go#L654) or [redis.NewClusterClient](https://github.com/redis/go-redis/blob/80c9f5bb777dd4e6393d014cffe9d28428ecf756/osscluster.go#L884)?

**Comment by brianluong** [comment]
only `NewClusterClient` worked for me
